docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -262,22 +298,35 @@ spring:
 [source,java]
 ----
 @Bean
-public Consumer<List<String>> consume() {
-    return list -> list.forEach(event -> LOGGER.info("New event received: '{}'",event));
+public Consumer<Message<String>> consume() {
+    return message -> {
+        Checkpointer checkpointer = (Checkpointer) message.getHeaders().get(CHECKPOINTER);
+        LOGGER.info("New message received: '{}', partition key: {}, sequence number: {}, offset: {}, enqueued time: {}",
+                message.getPayload(),
+                message.getHeaders().get(EventHubsHeaders.PARTITION_KEY),
+                message.getHeaders().get(EventHubsHeaders.SEQUENCE_NUMBER),
+                message.getHeaders().get(EventHubsHeaders.OFFSET),
+                message.getHeaders().get(EventHubsHeaders.ENQUEUED_TIME)
+        );
+
+        checkpointer.success()
+                .doOnSuccess(success -> LOGGER.info("Message '{}' successfully checkpointed", message.getPayload()))
+                .doOnError(error -> LOGGER.error("Exception found", error))
+                .subscribe();
+    };
 }
 
 @Bean
 public Supplier<Message<String>> supply() {
     return () -> {
         LOGGER.info("Sending message, sequence " + i);
-        return MessageBuilder.withPayload("\"Hello world"+ i++ +"\"").build();
+        return MessageBuilder.withPayload("Hello world, " + i++).build();
     };
 }
 ----
 ==== Samples
 
-Please refer to this https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud
--azure_4.0/eventhubs/spring-cloud-azure-stream-binder-eventhubs to learn how to
+Please refer to this https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0/eventhubs/spring-cloud-azure-stream-binder-eventhubs to learn how to
```

Comment:
please do not use the link directly, write it as below:
https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0/eventhubs/spring-cloud-azure-stream-binder-eventhubs[sample project]

Same for service bus